app/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,8 @@
   <color name="colorPrimary">#008577</color>
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
+  <color name="colorIvory">#FFFFF0</color>
+  <color name="colorConceptToolbar">#C55F45</color>
+  <color name="colorConceptToolbarDarker">#A6503A</color>
```

Comment:
Can we come up with a more specific name than 'darker'? Which aspect of the toolbar is having this color? Can that be included in the name?

Reply:
The color is for the very top heading that has time and wifi info. I made a darker orange concept toolbar color to match the pattern the home screen has with its green and darker green.